### PR TITLE
7902793: Fix IllegalMonitorStateException in JInternalFrameOperatorCloseTest

### PR DESCRIPTION
**Pull request by amresh-sahu**
IllegalMonitorStateException has been fixed.

---------
### Progress
- [x] Change must not contain extraneous whitespace
- [x] Commit message must refer to an issue
- [ ] Referenced JBS issue must only be used for a single change
- [x] Change must be properly reviewed (no review required)

### Issue
 * [CODETOOLS-7902793](https://bugs.openjdk.org/browse/CODETOOLS-7902793): Fix IllegalMonitorStateException in JInternalFrameOperatorCloseTest (**Task** - P3) ⚠️ Issue is already resolved. Consider making this a "backport pull request" by setting the PR title to `Backport <hash>` with the hash of the original commit. See [Backports](https://wiki.openjdk.org/display/SKARA/Backports).

### Reviewing
<details><summary>Using <code>git</code></summary>

Checkout this PR locally: \
`$ git fetch https://git.openjdk.org/jemmy-v2.git pull/3/head:pull/3` \
`$ git checkout pull/3`

Update a local copy of the PR: \
`$ git checkout pull/3` \
`$ git pull https://git.openjdk.org/jemmy-v2.git pull/3/head`

</details>
<details><summary>Using Skara CLI tools</summary>

Checkout this PR locally: \
`$ git pr checkout 3`

View PR using the GUI difftool: \
`$ git pr show -t 3`

</details>
<details><summary>Using diff file</summary>

Download this PR as a diff file: \
<a href="https://git.openjdk.org/jemmy-v2/pull/3.diff">https://git.openjdk.org/jemmy-v2/pull/3.diff</a>

</details>

### Webrev
[Link to Webrev Comment](https://git.openjdk.org/jemmy-v2/pull/3#issuecomment-729589590)

### CODE REVIEW

**Comment by amresh-sahu** [comment]
please review @shurymury  @akolarkunnu

**Comment by amresh-sahu** [comment]
Please review @akolarkunnu @shurymury

File: test/org/netbeans/jemmy/operators/JInternalFrameOperatorCloseTest.java [hunk 1]
```diff
@@ -96,11 +97,7 @@ public void internalFrameDeactivated(InternalFrameEvent e) {
 
             @Override
             public void internalFrameClosing(InternalFrameEvent e) {
-                try {
-                    this.wait(80000);
-                } catch (InterruptedException e1) {
-                    e1.printStackTrace();
-                }
+                    new QueueTool().waitEmpty();
```
Comment (shurymury) [inline test/org/netbeans/jemmy/operators/JInternalFrameOperatorCloseTest.java]: There is a comment below, which explains what should be here:
// Making not to close the fame for 1 minute and expecting TimeoutExpiredException
// from waitClosed()

Which would mean that the lines in internalFrameClosing(InternalFrameEvent e) should just delay the execution for  minute. The reason for doing that, it looks like, is that there is no other way to detect window closing but by the timeout.

Your fix is doing something completely different.

**Review by shurymury — CHANGES_REQUESTED** [review]

File: test/org/netbeans/jemmy/operators/JInternalFrameOperatorCloseTest.java (hunk 1 above)
Comment (amresh-sahu) [reply test/org/netbeans/jemmy/operators/JInternalFrameOperatorCloseTest.java]: Removed negative test scenario. 
Currently, Testing that Internal frame is closing or not.

**Pull request closed without merging**